src/SDL_shadercross.c

```diff
@@ -835,7 +835,6 @@ static void *SDL_ShaderCross_INTERNAL_CreateShaderFromHLSL(
         &bytecodeSize);
 
     if (spirv == NULL) {
-        SDL_SetError("%s", "Failed to compile SPIR-V!");
```

Comment:
Can you add a comment here along the lines of `Error has already been set`?

Reply:
Oh, yes good catch. Done.